packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx

```diff
@@ -33,8 +33,14 @@ const TabNavigationMenu = ({onClickSection, classes}: {
   classes: ClassesType,
 }) => {
   const currentUser = useCurrentUser();
+  const { captureEvent } = useTracking()
   const { TabNavigationItem, FeaturedResourceBanner } = Components
   const customComponentProps = {currentUser}
+  
+  const handleClick = (e, tabId) => {
+    captureEvent(`${tabId}NavClicked`)
```

Comment:
I only see a linkClicked event, not this one you added. I'm just clicking on the sidebar menu. Not sure if I'm doing something wrong?

![image](https://user-images.githubusercontent.com/221511/156225185-b5ed8178-edab-405b-bf6f-1dba8bea923b.png)

Reply:
Hmm I saw it in the console when I tested - I'll go ahead and deploy this, and if it's not working on prod I'll look into it more